samples/excel/85-preview-apis/worksheet-find-all.yaml

```diff
@@ -0,0 +1,154 @@
+order: 4
+id: excel-worksheet-find-all
+name: Find all ranges
+description: Finds data within a worksheet.
+host: EXCEL
+api_set:
+    ExcelAPI: 1.9
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#findCompleted").click(() => tryCatch(findCompleted));
+        $("#findDelayed").click(() => tryCatch(findDelayed));
+        $("#findCanceled").click(() => tryCatch(findCanceled));
+
+        async function findCompleted() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const foundRanges = sheet.findAllOrNullObject("Complete", {
+                    completeMatch: true,
+                    matchCase: false
+                });
+
+                await context.sync();
+
+                if (foundRanges.isNullObject) {
+                    console.log("No complete projects");
+                } else {
+                    foundRanges.format.fill.color = "green"
+                }
+            });
+        }
+
+        async function findDelayed() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const foundRanges = sheet.findAllOrNullObject("Delay", {
+                    completeMatch: false,
+                    matchCase: false
+                });
+
+                await context.sync();
+
+                if (foundRanges.isNullObject) {
+                    console.log("No delayed projects");
+                } else {
+                    foundRanges.format.fill.color = "yellow"
+                }
+            });
+        }
+
+        async function findCanceled() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                // NOTE: In this sample, there are no canceled projects. Using findAll (without the OrNullObject) would throw an ItemNotFound error.
```

Comment:
Consider rewording -> 

// NOTE: In this sample, there are no canceled projects. Calling sheet.findAll(...) instead of sheet.findAllOrNullObject(...) would throw an ItemNotFound error.